app/src/org/commcare/connect/ConnectConstants.java

```diff
@@ -44,4 +45,44 @@ public class ConnectConstants {
     public static final String CONNECT_KEY_SECONDARY_PHONE = "secondary_phone";
     public static final String CONNECT_KEY_VALIDATE_SECONDARY_PHONE_BY = "secondary_phone_validate_by";
     public static final String CONNECT_KEY_DB_KEY = "db_key";
+
+
+    public final  static int CONNECT_NO_ACTIVITY=ConnectConstants.ConnectIdTaskIdOffset + 1;
```

Comment:
Looks like a bunch of extra spaces in these lines between final and static. Easy to clean up with the help of the Alt key!